### ISSUE TITLE
Problems on Mac OSX with python 3

### ISSUE THREAD

**Issue by xabbu137** [regebro/tzlocal#30]
Hi - thanks for the code, but I have problems on Mac OSX (I am using 10.11.3) and python 3:

```
Python 3.5.1 (v3.5.1:37a07cee5969, Dec  5 2015, 21:12:44)
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import tzlocal
>>> tzlocal.get_localzone()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/axs/Envs/testtime/lib/python3.5/site-packages/tzlocal/darwin.py", line 29, in get_localzone
    _cache_tz = _get_localzone()
  File "/Users/axs/Envs/testtime/lib/python3.5/site-packages/tzlocal/darwin.py", line 16, in _get_localzone
    tzname = pipe.stdout.read().replace('Time Zone: ', '').strip()
TypeError: a bytes-like object is required, not 'str'
>>>
```

This seems to be specific to python 3, as it is working nicely on python 2.7:

```
Python 2.7.10 (v2.7.10:15c95b7d81dc, May 23 2015, 09:33:12)
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import tzlocal
>>> tzlocal.get_localzone()
<DstTzInfo 'Europe/Zurich' BMT+0:30:00 STD>
```

Thanks for your help!

**Comment by leonrodenburg** [comment]
See this PR for a fix: #29

**Comment by xabbu137** [comment]
Thanks for your quick help and fix!
